userguide/system.rst

```diff
@@ -394,16 +394,15 @@ Each boot environment entry contains this information:
   |ui-options| and :guilabel:`Keep` for an entry if that boot
   environment should not be automatically pruned.
 
-Click |ui-options| for an entry to view the configuration buttons for
-it.  These configuration buttons are shown:
+Click |ui-options| on an entry to see these configuration buttons:
 
 * **Delete:** used to delete the highlighted entry, which also removes
-  that entry from the boot menu. Since you cannot delete an entry that
-  has been activated, this button will not appear for the active boot
-  environment. If you need to delete an entry that  is currently
+  that entry from the boot menu. Since an activated entry 
+  cannot be deleted, this button will not appear for the active boot
+  environment. To delete an entry that is currently
   activated, first activate another entry, which will clear the
   *On reboot* field of the currently activated entry. Note that this
-  button will not be displayed for the *default* boot environment as
+  button will not appear for the *default* boot environment as
   this entry is needed in order to return the system to the original
   installation state.
 
```

Comment:
Change
Since an activated entry cannot be deleted, this button will not appear for the active boot environment.
to
Active boot environments cannot be deleted.
Since active boot environments cannot be deleted it'd make sense the Delete button would not appear.

s/is currently activated/is currently active/
I don't think activated should be used here.

Something for the future: Turn the note about the default boot environment into a real note callout and reword a tad.